tests/ui/target-feature/tied-features-no-implication-1.rs

```diff
@@ -5,11 +5,19 @@
 //@[paca] error-pattern: the target features paca, pacg must all be either enabled or disabled together
 //@[pacg] compile-flags: -Ctarget-feature=+pacg
 //@[paca] error-pattern: the target features paca, pacg must all be either enabled or disabled together
-#![feature(no_core, lang_items)]
+#![feature(no_core, lang_items, const_trait_impl)]
 #![no_core]
 
-#[lang="sized"]
-trait Sized {}
+#[lang = "pointeesized"]
+pub trait PointeeSized {}
+
+#[lang = "metasized"]
+#[const_trait]
+pub trait MetaSized: PointeeSized {}
+
+#[lang = "sized"]
+#[const_trait]
+pub trait Sized: MetaSized {}
```

Comment:
maybe these tests should be using minicore... well, whatever for now.

Reply:
I ported as many tests as I could to using minicore in rust-lang/rust#137599, if this one wasn't ported it was either because adding minicore made it fail or because adding minicore made it harder to debug/reason about/some-other-test-specific-reason.